### DISCUSSION TITLE
An error occurred while loading the media's data

### DISCUSSION THREAD

**Discussion by Dan1ell** [mediacms-io/mediacms#98] [Q&A] [answered]
After uploading a video, the process completes with the option to view the media.
However, the page to view the media shows: "An error occurred while loading the media's data".

Even though the videos are visible in the admin,
they are not visible on the home page nor in recent uploads.

I'm currently running MediaCMS on a server that I access through its IP address.
Ubuntu 18.04.1

What could be the issue?

**Comment by gam-ragnar** [comment]
@Dan1ell check your logs

**Reply by Dan1ell** [reply]
debug.log

> Invalid HTTP_HOST header: ''. The domain name provided is not valid according to RFC 1034/1035.

**Comment by mgogoulos** [comment, marked as answer]
If you can post a screenshot or some message from the browser debug console, could help to identify the issue 

Most probably related with https://github.com/mediacms-io/mediacms/issues/101

**Reply by Dan1ell** [reply]
After setting the FRONTEND_HOST to the server's ip address, the app started working correctly. I can now see my uploaded media and new uploads have no issues.
